resources/diff/rules/defaultRules.json

```diff
@@ -166,7 +166,7 @@
     }
   },
   {
-    "name": "Ignore all changes in examples",
+    "name": "Example Changed",
```

Comment:
same here- this rule ignores all changes to the example nodes

Reply:
This rule detects that an example has been changed. It is currently in the category "Ignored". None of the other rules have their category in the name, and I don't think that this should either.

Reply:
my only point is, the rule name should be descriptive enough so that we/user can understand the rule with out looking at all the conditions with in the rule

Reply:
Yes, and I think that "Example Changed" is descriptive of the conditions that trigger the rule. "Ignored" isn't part of that, it's part of how we handle the rule once triggered. And I think that the name should be a label for what the rule does, not how we use the result.

To put it another way, I don't see the category as fundamentally part of the rule. If we change the category, then I don't think the rule itself has changed. If the rule hasn't changed, then the name shouldn't need to be updated. Therefore, I think that the category should not be included in the name, otherwise the name would have to be updated even when the rule has not changed.